_posts/open-science-for-bioinformaticians

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Filename - two things needed: 
- add `.md` to the end - this is needed so the website knows to turn it into a post. 
- add a date to the start, e.g. `2024-20-21-open-science[etc...].md`

Reply:
Thanks, Yo. 🙏 

@seunolufemi123 - The naming convention is `[year]-[month]-[day]-[post-name].md.`